### DISCUSSION TITLE
%E8%AE%BA%E6%96%87%E8%A7%A3%E8%AF%BB/convnext/

### DISCUSSION THREAD

**Discussion by giscus** [thb1314/thb1314.github.io#1] [Announcements]
# %E8%AE%BA%E6%96%87%E8%A7%A3%E8%AF%BB/convnext/

http://127.0.0.1:8000/%E8%AE%BA%E6%96%87%E8%A7%A3%E8%AF%BB/convnext/

**Comment by thb1314** [comment]
自问自答，这是一条仅供参考的评论

**Comment by xcyuyuyu** [comment]
你好，请问你对比过convnext的pytorch版本与trt版本的速度吗，我直接转成trt，发现速度并没有变快

**Reply by thb1314** [reply]
请问有具体数据吗？我们可以一起看下

**Reply by thb1314** [reply]
第三章第二个表格中，其他人测试过，是由加速效果的，但是这个是需要看具体trt版本和cudnn版本
